recipes/cuda-version/meta.yaml

```
@@ -0,0 +1,33 @@
+{% set name = "cuda-version" %}
+{% set version = "12.0.0" %}
+{% set major_version = version.split('.')[0]|int %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
```

Maybe we can include the license file from here? Though open to other ideas

```suggestion

source:
  - url: https://gitlab.com/nvidia/headers/cuda/-/raw/master/LICENSE
```

---

Done below